src/apps/frontend/types/task.ts

```diff
@@ -0,0 +1,5 @@
+export interface Task {
+  id: string;
```

Comment:
nit: order alphabetically

In general, order properties alphabetically.

Also, I believe this might need to be a class, instead of interface. This would allow to convert json returned from server eventually into a JS object to work with on the client side.